projects/ppwcode/ng-common-components/src/lib/expandable-card/expandable-card.component.html

```diff
@@ -7,14 +7,14 @@
         (closed)="panelOpenState = canBeCollapsed"
     >
         <mat-expansion-panel-header class="ppw-expandable-panel-header">
-            <mat-panel-title>
+            <mat-panel-title class="ppw-expandable-panel-title">
```

Comment:
"card" als term gebruiken gebruiken in plaats van "panel"

Het component zelf heet ook "card"